pdc_client/plugins/compose_image_rtt_tests.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+import json
+
+from pdc_client import get_paged
+from pdc_client.plugin_helpers import PDCClientPlugin, add_parser_arguments, extract_arguments
+
+
+class ComposeImageRTTTestsPlugin(PDCClientPlugin):
+    def register(self):
+        self.set_command('compose-image-rtt-tests')
+        list_parser = self.add_action('list', help='list all compose image rtt tests')
+        add_parser_arguments(list_parser, {'compose': {},
+                                           'variant': {},
+                                           'arch': {},
+                                           'file_name': {},
+                                           'test_result': {}},
+                             group='Filtering')
+        list_parser.set_defaults(func=self.list_compose_image_rtt_tests)
+
+        info_parser = self.add_action('info', help='display details of a compose image RTT test')
+        self._add_common_arguments(info_parser)
+        info_parser.set_defaults(func=self.compose_image_rtt_test_info)
+
+        help_text = 'update an existing compose image\'s RTT test result (untested, passed or failed)'
+        update_parser = self.add_action('update', help=help_text)
+        self._add_common_arguments(update_parser)
+        update_parser.add_argument('test_result', metavar='TEST_RESULT')
```

Comment:
I think this format is not perfect, you can check other api. For example:
usage: pdc compose-image-rtt-tests update [-h]  [--test_result  TEST_RESULT]
                                          COMPOSE_ID VARIANT_UID ARCH   FILE_NAME